server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1971,21 +1972,26 @@ public Response bind(
         try {
             consumerBindUtil.validateServiceLevel(consumer.getOwner(), serviceLevel);
             dryRunPools = entitler.getDryRun(consumer, serviceLevel);
-            if (dryRunPools == null) {
-                return new ArrayList<PoolQuantity>();
-            }
         }
         catch (ForbiddenException fe) {
-            return dryRunPools;
+            return Collections.<PoolQuantityDTO>emptyList();
         }
         catch (BadRequestException bre) {
             throw bre;
         }
         catch (RuntimeException re) {
-            return dryRunPools;
+            return Collections.<PoolQuantityDTO>emptyList();
```

Comment:
We can skip returning from the exception handlers here and just fall through to the new translation block.